proto/batch_plan.proto

```diff
@@ -30,6 +30,11 @@ message RowSeqScanNode {
   }
   // If along with `batch_limit`, `chunk_size` will be set.
   ChunkSize chunk_size = 6;
```

Comment:
The `chunk_size` is also used for optimizing early exiting for Top-N and limit. Is there anyway to unify them?

Reply:
IIUC, if the downstream is responsive enough and there's no aggressive prefetching in Hummock, setting the `chunk_size` should be enough as the downstream Top-N or Limit will abort the scan's execution in time. 🤔

Reply:
Agree. Stopping prefetching for small limit scan is the main motivation.

Reply:
But to make this parameter `limit` in `RowSeqScan` self-contained, return early seems more appropriate.